d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -26,21 +27,27 @@ def on_market_cycle(self, market_info):
         if self.is_finished is True:
             return
 
-        market_uuid = self.get_uuid_from_area_name("Market")
+        market_uuid = self.get_uuid_from_area_name("House")
         for area_uuid, area_dict in self.latest_grid_tree_flat.items():
-            if "asset_info" not in area_dict or area_dict["asset_info"] is None:
-                continue
-
             logging.info(
-                f"current_market_fee: {self.grid_fee_calculation.calculate_grid_fee(area_uuid, market_uuid)}")
-            if key_in_dict_and_not_none_and_greater_than_zero(area_dict, "available_energy_kWh"):
-                self.add_to_batch_commands.offer_energy(area_uuid=area_uuid, price=1,
-                                                        energy=area_dict["asset_info"]["available_energy_kWh"] / 2) \
-                    .list_offers(area_uuid=area_uuid)
-            if key_in_dict_and_not_none_and_greater_than_zero(area_dict, "energy_requirement_kWh"):
-                self.add_to_batch_commands.bid_energy(area_uuid=area_uuid, price=30,
-                                                      energy=area_dict["asset_info"]["energy_requirement_kWh"] / 2) \
-                    .list_bids(area_uuid=area_uuid)
+                f"current_market_fee: "
+                f"{self.grid_fee_calculation.calculate_grid_fee(area_uuid, market_uuid)}")
+            if "asset_info" not in area_dict or area_dict["asset_info"] is None:
+                if area_uuid == market_uuid:
+                    self.add_to_batch_commands.last_market_dso_stats(area_uuid=area_uuid). \
+                        grid_fees(area_uuid=area_uuid, fee_cents_kwh=5)
+            else:
+                if key_in_dict_and_not_none_and_greater_than_zero(area_dict["asset_info"],
+                                                                  "available_energy_kWh"):
+                    energy = area_dict["asset_info"]["available_energy_kWh"] / 2
+                    self.add_to_batch_commands.offer_energy(area_uuid=area_uuid, price=1,
+                                                            energy=energy)
+
+                if key_in_dict_and_not_none_and_greater_than_zero(area_dict["asset_info"],
+                                                                  "energy_requirement_kWh"):
```

Comment:
This is unrelated to the current PR: I've seen this repeated in several scripts. Maybe it would help if we created a separate method that makes the intention more explicit. Something like: 
```suggestion
                if self.has_available_energy(area_dict):
```
and the checks happen inside the method.

Reply:
Yes, I reused your `_can_place_bid` and `_can_place_offer` in the integrationtests, but not here.
I agree, that it makes the intention more clear. 
We should review all `test_*` scripts in `/setups` and think about how user-friendly they are. For now i thought these are only for us for testing, but really they should be "good" examples. 
Do you think this is worth a separate task?

Reply:
Actually, I think this is what happens in `test_aggregator_ess.py` with `_can_place_offer` and `_can_place_bid`.

Reply:
> We should review all test_* scripts in /setups and think about how user-friendly they are. For now i thought these are only for us for testing, but really they should be "good" examples.
Do you think this is worth a separate task?

Yes, I think you're definitely right. But I wouldn't do this in the current PR: having a separate task as you suggest would be better in my opinion.